doc/releases/pacific.rst

```diff
@@ -2,6 +2,209 @@
 Pacific
 =======
 
+v16.2.14 Pacific
+================
+
+This is the fourteenth backport release in the Pacific series.
+
+Notable Changes
+---------------
+
+* CEPHFS: After recovering a Ceph File System post following the disaster recovery
```

Comment:
CephFS

Reply:
We've always used both: CephFS / CEPHFS - its a mix

Reply:
Not an acronym :(

Reply:
There is a only a single instance of CEPHFS in the release notes (and all of `doc`!), which sneaked in by accident.  Not to bike shed, but it should be CephFS.

Reply:
Hmm.. I do recall seeing more than an instance of it - would have been in a different branch.

Reply:
@yuriw Please address Anthony's nit (CEPHFS -> CephFS).